program/src/entrypoint-runtime-verification.rs

```diff
@@ -209,6 +209,42 @@ fn get_rent(_account_info: &AccountInfo) -> solana_rent::Rent {
 // fn cheatcode_is_multisig(_: &AccountInfo) {}
 // fn cheatcode_is_rent(_: &AccountInfo) {}
 
+// special test for basic domain data access (SPL types)
+#[inline(never)]
+fn test_ptoken_domain_data(acc: &AccountInfo, mint: &AccountInfo, rent: &AccountInfo) {
```

Comment:
Maybe change the name so we don't confuse it with the other?
```suggestion
fn test_spltoken_domain_data(acc: &AccountInfo, mint: &AccountInfo, rent: &AccountInfo) {
```